ymcatwincities/openy_activity_finder
# Implement "limit by category" in AF4

https://github.com/ymcatwincities/openy_activity_finder/pull/20 to work on #7  added a "Limit by category" field to the AF paragraph but didn't implement it. This implements that functionality as a step toward #103. Once we've validated that the logic here works with the Solr backend, then I'll continue work toward generalizing the functionality.

To test:

- Enable openy_prgf_af4_demo and ensure solr is indexed
- edit the AF4 paragraph and add an item to the "Limit by category" field.
- observe AF is limited to only the selected categories
- edit again and set the "Exclude by category"
- retest and observe Exclude works as expected.
- optional: it is technically possible to have both limit and exclude set, but in almost all circumstances this will result in an empty result.
